src/main/java/com/weather3d/CyclesPlugin.java

```diff
@@ -994,72 +996,90 @@ private void syncBiome()
 
 	private void syncSeason()
 	{
-		if (config.themes117())
-		{
-			Collection<Plugin> plugins = pluginManager.getPlugins();
+		boolean is117Enabled = pluginManager.getPlugins().stream()
+			.filter((plugin) -> plugin.getName().equals("117 HD"))
+			.map((plugin) -> pluginManager.isPluginEnabled(plugin)).findFirst().orElse(false);
 
-			for (Plugin plugin : plugins)
+		if (config.themes117() && is117Enabled)
```

Comment:
In the event the config isn't set or 117 isn't available we're just going to  fallback to the base functionality of 3D Weather